### PR TITLE
fix: add version in app and api

### PR DESCRIPTION
**Pull request by esoubiran-aneo**

### CODE REVIEW

File: apps/api/src/app/app.controller.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export class AppController {
    * Used to view a message to check if the API is up and running.
    */
   @Get()
-  index(): { message: string } {
-    return { message: 'Welcome to ArmoniK GUI API' };
+  index() {
```
Comment (lemaitre-aneo) [inline apps/api/src/app/app.controller.ts:9]: Why the return type has been removed?

File: scripts/update-version.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    exit 1
+fi
+
+# If -h or --help is passed, failed
+if [ "$1" == "-h" ] || [ "$1" == "--help" ]; then
+    echo "Usage: $0 <version>"
+
+    echo "This script will update the version in all the projet using the argument as the version."
+    echo "The version follow semver format (eg: 1.0.0)."
+    exit 1
```
Comment (lemaitre-aneo) [inline scripts/update-version.sh:16]: help should not be an error

File: apps/api/src/app/app.controller.ts (hunk 1 above)
Comment (esoubiran-aneo) [reply apps/api/src/app/app.controller.ts]: j'ai tout bien remis

File: scripts/update-version.sh (hunk 2 above)
Comment (esoubiran-aneo) [reply scripts/update-version.sh]: bien vu, c'est fix

File: scripts/update-version.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+if [ "$1" == "-h" ] || [ "$1" == "--help" ]; then
+    echo "Usage: $0 <version>"
+
+    echo "This script will update the version in all the projet using the argument as the version."
+    echo "The version follow semver format (eg: 1.0.0)."
+    exit 0
+fi
+
+# Replace version in package.json file
+sed -i "s/\"version\": \".*\"/\"version\": \"$1\"/g" package.json
```
Comment (lemaitre-aneo) [inline scripts/update-version.sh:20]: To be clean, I would replace `.*` with `[^\"]*`

File: scripts/update-version.sh [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+echo "Version updated in package.json"
+
+# Replace version in app.controller.ts file
+sed -i "s/version: '.*'/version: '$1'/g" apps/api/src/app/app.controller.ts
```
Comment (lemaitre-aneo) [inline scripts/update-version.sh:24]: To be clean, I would replace `.*` with `[^']*`

**Review by lemaitre-aneo — APPROVED** [review]

File: scripts/update-version.sh (hunk 4 above)
Comment (esoubiran-aneo) [reply scripts/update-version.sh]: done

**Pull request merged**